src/app/data-explorer/data-class-row/data-class-row.component.html

```diff
@@ -64,15 +66,15 @@ <h3>{{ (dataClassWithElements?.dataClass)!.label }}</h3>
       </button>
     </span>
   </div>
-  <div [hidden]="!expanded" class="mdm-my-request__request-elements">
+  <div [hidden]="!expanded" class="mdm-my-data-specification__specification-elements">
```

Comment:
The `mdm-my-data-specification__specification-elements` class is located in the `my-data-specification-detail.component.scss` file. But you're correct that there are some unused css classes floating around. I decided to keep this task to strictly renaming rather than additional refactoring for the sake of expediency.